components/CourseViewer.tsx

```diff
@@ -66,6 +66,11 @@ const CourseViewer = ({ course, lessonProgress = [], setLessonProgress }: Props)
             playbackId={playbackId}
             placeholder={placeholder}
             onEnded={markLessonCompleted}
+            metadata={{
+              video_series: activeLesson.courseId,
+              video_title: activeLesson.name,
+              player_name: "Video Course Starter Kit",
```

Comment:
question(non-blocking): Any reason you included "Mux Player" in the name for stream.new but are not including it here?

Reply:
No reason, I think I'm going to change it on stream.new side actually, cause Mux Player will come through the `player_software_name`. Good call